examples/replit_token_storage.py

```diff
@@ -0,0 +1,22 @@
+import nextcord
+from nextcord.ext import commands
+import os
```

Comment:
``import os`` should be the first line, followed by a line gap then the nextcord imports inline with pep8